sys/geom/geom_subr.c

```diff
@@ -1641,7 +1645,7 @@ DB_SHOW_COMMAND(bio, db_show_bio)
 	struct bio *bp;
 
 	if (have_addr) {
-		bp = (struct bio *)addr;
+		bp = DB_DATA_PTR(addr, sizeof(*bp));
```

Comment:
We could perhaps have a `DB_DATA_PTR_OBJ` that does `sizeof(*arg)`.  Many uses of `DB_DATA_PTR` don't have a typed variable.